### ISSUE TITLE
Oneplus 3 issue

### ISSUE THREAD

**Issue by muschiu40jp** [ShadySquirrel/ViPER4Android#40]
Custom rom Resurrection Remix Android Oreo 8.1 other modules Enable doze for GMS and Xposed framework. No other sound module installed.
The drive is installed and shows no and processing also no. I did dirty update to the custom Rom and everytime I do updates is coming with Audio Fx installed but even after i uninstalled the module is not working. And when it was work and enabled Bluetooth no Bluetooth headphones works.

**Comment by ShadySquirrel** [comment]
This was discused many times so far and looks like some Op3 specific bug - combination of factors probably - f2fs, encryption and Magisk/sepolicy.
You can try with newest version where I've finally moved sepolicy to late_start scripts instead of post-fs.

**Comment by muschiu40jp** [comment]
Ok i will try.I just saw that a new version has been release.

**Comment by muschiu40jp** [comment]
Ok i will try.I just saw that a new version has been release.

**Comment by ShadySquirrel** [comment]
Please attach config file and full logcat. Try with permissive sepolicy, too

**Comment by muschiu40jp** [comment]
I have tried with changing selinux to permissive and has work but then i restarted again and again with selinux set to enforcing is not working. Here is a requested files.

[full logcat.txt](https://github.com/ShadySquirrel/ViPER4Android/files/1962662/full.logcat.txt)

**Comment by ShadySquirrel** [comment]
Pack them in a zip :) I'll take a look at logcat later.

**Comment by muschiu40jp** [comment]
Ok i will send you a logcat from desktop now i just archive audio config files 
[Audio Config.zip](https://github.com/ShadySquirrel/ViPER4Android/files/1963591/Audio.Config.zip)

**Comment by ShadySquirrel** [comment]
Can you try with new module I've pushed today? There was an error on my side.

**Comment by ShadySquirrel** [comment]
No, that's not a device problem, generally V4A has issues with Bluetooth, I saw that people are complaining, but I can't create a test case because I don't have wireless headphones.
